data/sql_updates/create_candidate_detail_view.sql

```diff
@@ -54,6 +54,14 @@ from dimcand
         select distinct on (cand_sk) * from dimcandproperties
             order by cand_sk, candproperties_sk desc
     ) cand_p_most_recent using(cand_sk, cand_id)
+    -- there are some holes in this data so we want the lat time it was updated
+    left join(
+        select distinct on (cand_sk)
+            s.cand_sk, s.cand_ici_desc, s.cand_ici_cd
```

Comment:
on a related note. I used to use the following to have a test data set locally:

psql -f data/cfdm_test.pgdump.sql cfdm_test

Did that change or do we not have test data anymore?